src/alert-debuglog.c

```diff
@@ -178,6 +178,9 @@ static TmEcode AlertDebugLogger(ThreadVars *tv, const Packet *p, void *thread_da
     } else if (PKT_IS_IPV6(p)) {
         PrintInet(AF_INET6, (const void *)GET_IPV6_SRC_ADDR(p), srcip, sizeof(srcip));
         PrintInet(AF_INET6, (const void *)GET_IPV6_DST_ADDR(p), dstip, sizeof(dstip));
+    } else {
+        snprintf(srcip, sizeof(srcip), "unknown-ip-version");
```

Comment:
this is dead code, since there is just one path that can get us here and that checks for ip4/ip6. I prefer to assist static code analysis with assertions instead

Reply:
ok, was not sure it was dead code :-)